sqlmodel/_compat.py

```diff
@@ -64,6 +66,35 @@ def _is_union_type(t: Any) -> bool:
 finish_init: ContextVar[bool] = ContextVar("finish_init", default=True)
 
 
+def set_polymorphic_default_value(
+    self_instance: _TSQLModel,
+    values: Dict[str, Any],
+) -> bool:
+    """By defalut, when init a model, pydantic will set the polymorphic_on
```

Comment:
Very minor typo here

```suggestion
    """By default, when init a model, pydantic will set the polymorphic_on
```